src/main/resources/default/assets/tycho/styles/forms.scss

```diff
@@ -137,23 +137,20 @@
 }
 
 .dropzone.sirius-imageupload {
+  height: 350px;
+
   .img-preview {
```

Comment:
This isn't idiomatic to bootstrap4. We should remove this class and add the appropriate marker css classes to the img / div itself like d-flex justify-content-center flex-grow-1 ....